portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,59 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getDoja() {
+  console.log('Fetching a doja cat lyric');
+
+  // The fetch() function returns a Promise because the request is asynchronous.
+  const responsePromise = fetch('/data');
+
+  // When the request is complete, pass the response into handleResponse().
+  responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
+  console.log('Handling the response.');
+
+  // response.text() returns a Promise, because the response is a stream of
+  // content and not a simple variable.
+  const textPromise = response.text();
+
+  // When the response is converted to text, pass the result into the
+  // addQuoteToDom() function.
+  textPromise.then(addQuoteToDom);
+}
+
+/** Adds a random quote to the DOM. */
+function addQuoteToDom(test) {
+  console.log('Adding quote to dom: ' + test);
+
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = test;
+}
+
+function getServerStats() {
```

Comment:
do you have two methods that are retrieving from the same endpoint and showing all of the lyrics? It looks like this method serves the same purpose as "getDoja"